stories/Atom/BaseTypography/Abbr/abbr.scss

```diff
@@ -1,6 +1,6 @@
-@import '../../../assets/scss/mixins';
-@import '../../../assets/scss/breakpoints';
-@import '../../../assets/scss/variables';
+@import url('../../../assets/scss/mixins');
```

Comment:
Similar question here, what's the need to change to use the `url(` import syntax?

Reply:
I haven't worked on a code change here as well: screenshot of my code: all my works are on the clean-up branch

<img width="1085" alt="Screenshot 2023-01-06 at 10 56 29 AM" src="https://user-images.githubusercontent.com/12831545/210998967-10f155fc-aa3a-428e-96ac-e58a491a6317.png">

Reply:
If you look at the `main` branch, you'll see that the code doesn't have the `url(` on the imports. I'm confused as to why this is in your PR?

Reply:
Abbreviations were not part of the items to remove so I didn't work on them. only 1 item was removed from the Atoms section:  Atom ->ProjectColumn